how-to/manage-users.rst

```diff
@@ -41,14 +41,17 @@ user by running ``sudo -iu stg-launchpad``.
 2. SSH into the ``launchpad-scripts`` juju unit, by running 
 ``in-model qastaging juju ssh launchpad-scripts/leader``.
 
-3. Add the user to the team by running: 
+3. Switch to ``launchpad`` user, by running 
+``sudo su launchpad``.
+
+4. Add the user to the team by running: 
 
 .. code::
      
-     /srv/launchpad/code/utilities/anoint-team-member <username> <team-name>
+     LPCONFIG=launchpad-scripts /srv/launchpad/code/utilities/anoint-team-member <username> <team-name>
```

Comment:
This needs to be changed to `LPCONFIG=launchpad-admin` if we are going to document doing this step on the `launchpad-admin` unit.